core/src/banking_stage.rs

```diff
@@ -822,6 +824,42 @@ impl BankingStage {
         }
     }
 
+    fn make_decision(
```

Comment:
Eventually this will go into the scheduler - separate it out now to make the transition simpler.